README.md

```diff
@@ -19,7 +20,9 @@ and/or projects resulting in the production of large amounts of data, e.g. seque
 
 The backend interface is built using [Flask](https://flask.palletsprojects.com/en/2.0.x/).
 
+---
 ## Development
+<br>
```

Comment:
Not convinced about using `<br>` in markdown is a good idea, but it looks good on the github rendering so ok then.

Reply:
I thought the page was too crowded and a bit difficult to get an overview of without any spaces, but I don't know any other way to do spaces in markdown.